README.md

```diff
@@ -110,14 +110,13 @@ For `production` mode, the `outputStyle` (old API) and `style` (new API) options
 
 Webpack provides an [advanced mechanism to resolve files](https://webpack.js.org/concepts/module-resolution/).
 
-The `sass-loader` uses Sass's custom importer feature to pass all queries to the Webpack resolving engine.
-Thus you can import your Sass modules from `node_modules`.
+The `sass-loader` uses Sass's custom importer feature to pass all queries to the webpack resolving engine enabling you to import your Sass modules from `node_modules`.
 
 ```scss
 @import "bootstrap";
 ```
 
-Using `~` is deprecated and can be removed from your code (**we recommend it**), but we still support it for historical reasons.
+Using `~` is deprecated and should be removed from your code (**we recommend it**), but we still support it for historical reasons.
```

Comment:
should be removed, but we still recommend it? Hmm.  I think you can rephrase this to `could`be removed.

Reply:
it's saying that we recommend _removing it_, hence why the use of "should" here - this sentence (like a number of others) could definitely be improved further, but I'm wanting to avoid taking on too much in the one PR for bandwidth reasons and I think it's worth landing the low hanging wins before attempting any bigger changes :)

Reply:
tbh frankly I think it's probably best short-term at least to just remove the "recommendation" - the fact that it's deprecated already signals that it's recommended to use alternatives, and this just confuses things further 🤷